### DISCUSSION TITLE
Need to change my SSO

### DISCUSSION THREAD

**Discussion by matheusgaya** [deepgram/community#415] [General help] [answered]
Hello!

My company used the Google SSO, and now changed to microsoft SSO.
With that, I cannot access my account anymore.
<img width="396" alt="image" src="https://github.com/deepgram/community/assets/35924490/05d13e75-d956-4854-a29f-18c0b880ce4a">

Could you please reset my user?
If i try to reset my password, its unavailable, because it says that i have logged in with Google SSO.
email: matheus.scandiffio@hyperflow.global.

Thanks!

**Comment by fakemonster** [comment, marked as answer]
Hey Matheus, we aren't able to transfer SSO providers like that, but you can still sign up with a new account using Azure SSO. We just sent an invite for you to rejoin your project, so if you sign up with Azure and follow that invite link you should be back on track!
